patches/llvm-project/0004-xfail-two-remaining-libcxx-with-picolibc-tests.patch

```diff
@@ -29,15 +29,43 @@ index 5a70ea5bd570..92dfbe8eb1ab 100644
 --- a/libcxx/test/std/language.support/support.start.term/quick_exit.pass.cpp
 +++ b/libcxx/test/std/language.support/support.start.term/quick_exit.pass.cpp
 @@ -16,6 +16,9 @@
- 
+
```

Comment:
I'm suspicious that this patch file seems to have had trailing whitespace stripped. Are you sure it still applies correctly like this? Normally I expect that the leading space on a patch line (indicating "neither added nor removed") is mandatory even when the rest of the line is empty. A _completely_ empty line in a patch file is a syntax error.

Same goes for the `-- ` sig-separator at the end of the patch file, added by `git format-patch`.

Reply:
I have edited this patch in an IDE. I belive it changed the whitespaces. However on linux patch applies without issues.

Reply:
I can confirm that trailing spaces were removed by IDE upon save of a file. I have enabled file.TrimTrailingWhitespace option.